crates/api_models/src/routing.rs

```diff
@@ -801,6 +802,7 @@ pub struct SuccessBasedRoutingConfigBody {
     pub default_success_rate: Option<f64>,
     pub max_aggregates_size: Option<u32>,
     pub current_block_threshold: Option<CurrentBlockThreshold>,
+    pub specificity_level: Option<SuccessRateSpecificityLevel>,
```

Comment:
Instead of making this optional and manually filling this field for the old entries, we can make this a required field but derive a `#[serde(default)]` attribute on top so that when getting deserialised, serde will take care of setting this to a default value instead of you doing it 
```suggestion
    #[serde(default)]
    pub specificity_level: SuccessRateSpecificityLevel,
```